js/index.js

```diff
@@ -2,22 +2,31 @@
 
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  let price = 0;
```

Comment:
How about declaring price and assigning the innerText of the query selectors ".price span" to it? In that case, could a const be used instead of a let?